### PR TITLE
adapt instructions.md for CNTI

### PR DESCRIPTION
**Pull request by martin-mat**
Refs #74

## Pre-submission checklist:

Please check each of these after submitting your pull request:

- [ ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/cnf-certification/blob/main/Certified_CNF_Form.md)?
- [ ] Did you include the product/project logo in SVG, EPS or AI format?
- [ ] Does your logo clearly state the name of the product/project/org and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
- [ ] If your product/project is open source, did you include the repo_url?
- [ ] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

File: instructions.md [hunk 1]
```diff
@@ -83,13 +83,13 @@ This file describes your product. It is YAML formatted with the following root-l
...
+| `vendor`            | Name of the legal entity that is certifying. This entity must have a signed participation form on file with the LFN  |
 | `name`              | Name of the product being certified. |
 | `version`           | The version of the product being certified. |
 | `website_url`       | URL to the product information website |
 | `repo_url`          | If your product is open source, this field is necessary to point to the primary GitHub repo containing the source. It's OK if this is a mirror. OPTIONAL  |
 | `documentation_url` | URL to the product documentation. OPTIONAL |
-| `product_logo_url`  | URL to the product's logo, (must be in SVG format -- not a PNG -- and include the product name). If not supplied, we'll use your company logo. Please see logo [guidelines](https://github.com/cncf/landscape#logos). |
+| `product_logo_url`  | URL to the product's logo, (must be in SVG format -- not a PNG -- and include the product name). If not supplied, we'll use your company logo. (TODO: LFN logo guidelines not found) Please see logo [guidelines](https://github.com/cncf/landscape#logos). |
```
Comment (taylor) [inline instructions.md:92]: @martin-mat @lixuna @Smitholi67 Maybe these guidelines can be copied over to the [cnti certification repo](https://github.com/lfn-cnti/certification) or the [cnti main repo](https://github.com/lfn-cnti/cnti).  Either way that can be a new pull request

**Review by taylor — APPROVED** [review]
lgtm

File: instructions.md [hunk 2]
```diff
@@ -24,16 +24,16 @@ There are [57 tests](docs/CNFCertification-1.0-beta.md) included in the CNF Cert
...
-Kind is one option tested and supported by the CNF Test Suite team.  Follow the [kind install](https://github.com/cncf/cnf-testsuite/blob/main/KIND-INSTALL.md) instructions to setup a cluster in [kind](https://kind.sigs.k8s.io/)
+Kind is one option tested and supported by the CNTI Test Catalog team.  Follow the [kind install](https://github.com/cnti-testcatalog/testsuite/blob/main/KIND-INSTALL.md) instructions to setup a cluster in [kind](https://kind.sigs.k8s.io/)
 
 
 ## Running
-The standard tool for running these tests is the [CNF Test Suite](https://github.com/cncf/cnf-testsuite). 
+The standard tool for running these tests is the [CNTI Test Catalog](https://github.com/cnti-testcatalog/testsuite). 
 
-1. Download the [certification binary release](https://github.com/cncf/cnf-testsuite/releases/tag/v0.29.1) of the test suite
+1. Download the [certification binary release](https://github.com/cnti-testcatalog/testsuite/releases/tag/v0.29.1) of the test suite
```
Comment (lixuna) [inline instructions.md:33]: the binary release URL will need to be updated for CNTI Certification v2.0 - release number pending

Comment (martin-mat) [reply instructions.md:33]: This should be updated along with 2.0 release changes in a dedicated PR.

**Comment by martin-mat** [comment]
Logo instruction left to CNCF and not intended to adapt it in this PR. Created a new issue to address it subsequemntly (#94)

**Review by taylor — APPROVED** [review]
lgtm

**Pull request merged**